### PR TITLE
[Feature] Login Message

### PR DESCRIPTION
**Pull request by moviebr**
### Changes Proposed

- Added the possibility to "close" the server with a custom warning.

### CODE REVIEW

File: src/configmanager.cpp [hunk 1]
```diff
@@ -238,6 +239,7 @@ bool ConfigManager::load()
 	string[URL] = getGlobalString(L, "url", "");
 	string[LOCATION] = getGlobalString(L, "location", "");
 	string[WORLD_TYPE] = getGlobalString(L, "worldType", "pvp");
+	string[BLOCK_LOGIN_TEXT] = getGlobalString(L, "blockLoginText", "pvp");
```
Comment (omarcopires) [inline src/configmanager.cpp:242]: I think the default message shouldn't just be "pvp" xD

**Comment by omarcopires** [comment]
It's a great feature, but it would be more interesting if it allowed players with roles to login.

File: src/configmanager.cpp (hunk 1 above)
Comment (moviebr) [reply src/configmanager.cpp:242]: haha I forgot to change it, soon I'll make a commit to change it

**Comment by marciNekel** [comment]
Wouldn't it be better just to change GAME_STATE_CLOSED server message to customizable one?

**Pull request closed without merging**